TravelBuddies/TravelBuddies/Models/ProfileViewController.m

```diff
@@ -35,15 +37,38 @@ - (void)viewDidLoad {
     
 }
 
-/*
-#pragma mark - Navigation
+-(void)getTimeline {
+
+    PFQuery *query = [PFQuery queryWithClassName:@"Post"];
+    [query orderByDescending:@"createdAt"];
+    query.limit = 20;
+    [query includeKey:@"author"];
+
+    // fetch data asynchronously
+    [query findObjectsInBackgroundWithBlock:^(NSArray *posts, NSError *error) {
+        if (posts != nil) {
+            self.posts = (NSMutableArray *)posts;
+            [self.collectionView reloadData];
+        } else {
+            NSLog(@"%@", error.localizedDescription);
+        }
+        //[self.refreshControl endRefreshing];
+    }];
+}
```

Comment:
If you have this function repeated in many classes, maybe it can be refactored later in an API manager